.github/dplasma/action.yml

```diff
@@ -0,0 +1,58 @@
+name: DPLASMA
+description: Uses the DPLASMA CI as a tester for PaRSEC
+
+jobs:
+  runs:
+    env:
+      BUILD_TYPE:  Release
+      SHARED_TYPE: ON
+      DEVICE: cpu
+      BUILD_DIRECTORY: ${{github.workspace}}/dplasma/build
+      INSTALL_DIRECTORY: ${{github.workspace}}/dplasma/install
+      SETUP: ${{github.workspace}}/dplasma/.github/workflows/env_setup.sh
+    defaults:
+      run:
+        shell: bash
+
+    steps:
+    - name: Setup Build Environment
+      run: |
+          git clone https://github.com/ICLDisco/dplasma
+          cd dplasma
+          rm -rf parsec
+          ln -s .. parsec
```

Comment:
You're replacing the internal parsec from dplasma with another one. Where is this parsec coming from ?

Reply:
The parsec will be the one that the current commit/PR is checking, but you can ignore all this for now. I'm trying to implement the dplasma  tests as reusable workflows or as real github actions so that there is less redundant code. I need to do some experiments to figure out the best way to do it though.

Reply:
then please keep this in draft mode until ready for reviews.

Reply:
It has been labeled with "draft".  Is there something else that I need to do?

Reply:
I did it with the github draft mode. That's all it needs to be, until ready for review.